src/LiveComponent/doc/index.rst

```diff
@@ -625,10 +625,33 @@ persisted entities, which dehydrate to an ``id``).
 Hydration, DTO's & the Serializer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you try to use a ``LiveProp`` for some unsupported type (e.g.a DTO object),
-it will fail. A best practice is to use simple data.
+.. versionadded:: 2.11
+
+    Dto are now a supported type for LiveComponent
+
+To use a ``LiveProp`` with DTO simply type properly your property
+
+.. code-block:: php
+
+    class ComponentWithAddressDto
+    {
+        public AddressDto $addressDto;
+    }
+
+And to work with DTO collection, you simply need to specified the collection type in your phpdoc
+
+.. code-block:: php
+
+    class ComponentWithAddressDto
+    {
+        /**
+         * @var AddressDto[]
+        /*
+        public array $addressDtoCollection;
+    }
 
-But there are two options to make this work:
+If this solution doesn't feat your need
+there are two others options to make this work:
```

Comment:
Before we finish with these last two lines, we should explain briefly how the DTO system works (please correct any inaccuracies in my statement below)

* It finds all properties on your DTO that are readable and writable and dehydrates each one.
* the PropertyAccess component is used, which means getter and setter methods are supported, in addition to public properties.
* The DTO cannot have any constructor arguments.